Cargo.toml

```diff
@@ -8,6 +8,17 @@ name = "cross"
 repository = "https://github.com/cross-rs/cross"
 version = "0.2.1"
 edition = "2021"
+include = [
+  "src/**/*",
+  "docker/Dockerfile.*",
+  "docker/*.sh",
```

Comment:
this isn't really needed, only the Dockerfiles need to be included, we should fix that later, as currently Dockerfile.x86_64-unknown-linux-gnu.centos is included as a builtin as well.

Reply:
Do we even need the Dockerfiles? People aren't building docker images from the registry, and if they were, the scripts would be needed. We don't reference them in the documentation currently either.